cfpb/consumerfinance.gov
# Added Sidebar and Expandable Button Translations for LEP pages

## Changes

- Added translation files and translation text for Arabic, Haitian Creole, Korean, Russian, Tagalog, Vietnamese, and Chinese languages.
- Languages missing "Show" and "Hide" translations: Korean, Russian, Vietnamese, Chinese

## How to test this PR

1. Refresh latest data with `./refresh-data.sh`
2. Run locally
3. Go to `http://127.0.0.1:8000/admin/pages/52/`
4. Click "View Draft" for each language and notice translations for the sidebar and "Show/Hide" `expandables` button (excluding aforementioned languages)

## Screenshots

<img width="389" alt="image" src="https://user-images.githubusercontent.com/85513449/216431481-fc46f607-b552-435f-8df7-4438606bfe9f.png">

<img width="379" alt="image" src="https://user-images.githubusercontent.com/85513449/216431548-ca8126ac-18e8-461b-933f-9cee6602253b.png">

<img width="393" alt="image" src="https://user-images.githubusercontent.com/85513449/216431609-e81004ca-b8ff-4e89-95b8-edf644043000.png">

## Future Work

- The spacing and layout of the sidebar does not account for languages other than English and needs to be adjusted
- Email address placeholder text needs to be translated into other languages
- Show/Hide buttons need translations for Korean, Russian, Vietnamese, Chinese
